File: uri/strings/the_library_of_mr_severino/2137.cpp
#include <iostream>
#include <vector>
#include <string>
#include <algorithm>

using namespace std;

int main() {
    int n;
    
    while (cin >> n) {
        vector<string> codes(n);
        
        for (int i = 0; i < n; i++) {
            cin >> codes[i];
        }
        
        sort(codes.begin(), codes.end());
        
        for (const string& code : codes) {
            cout << code << endl;
        }
    }
    
    return 0;
}

File: uri/strings/the_library_of_mr_severino/2137.rs
use std::io;

fn main() {
    let mut input = String::new();
    
    loop {
        input.clear();
        if io::stdin().read_line(&mut input).is_err() || input.trim().is_empty() {
            break;
        }
        
        let n: usize = match input.trim().parse() {
            Ok(num) => num,
            Err(_) => break,
        };
        
        let mut codes = Vec::new();
        for _ in 0..n {
            input.clear();
            io::stdin().read_line(&mut input).unwrap();
            codes.push(input.trim().to_string());
        }
        
        codes.sort();
        
        for code in codes {
            println!("{}", code);
        }
    }
}
